ImperatorToCK3Tests/CK3WorldTests/Province/CK3ProvincesTests.cpp

```diff
@@ -15,10 +15,10 @@ TEST(CK3World_CK3ProvincesTests, provincesAreProperlyLoadedFromFile)
 	ASSERT_EQ(4, provinces.getProvinces().size());
```

Comment:
Test is failing on appveyor here.

Reply:
All the failing tests are related to txt files, but the tests work fine locally. Have you experienced such issue with appveyor before?

Reply:
Only when I've had leftover local copies of the test files that it turned out the tests still expected in the old places.

Reply:
Correction: the tests also fail when I run them via ImperatorToCK3Tests.exe in Release folder locally.

Reply:
Maybe it needs to be $(OutDir)/whatever instead of $(Outdir)whatever

Reply:
The slash is at the end of OutDir: `$(ProjectDir)$(Configuration)\`